client/device/cloudftd/cloudftdonboarding/create.go

```diff
@@ -0,0 +1,136 @@
+package cloudftdonboarding
+
+import (
+	"context"
+	"fmt"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device/cloudfmc"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device/cloudfmc/fmcconfig"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device/cloudfmc/fmcplatform"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device/cloudftd"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/retry"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/user"
+	"time"
+)
+
+type CreateInput struct {
+	FtdUid string
+}
+
+func NewCreateInput(ftdId string) CreateInput {
+	return CreateInput{
+		FtdUid: ftdId,
+	}
+}
+
+type CreateOutput = fmcconfig.CreateDeviceRecordOutput
+
+func Create(ctx context.Context, client http.Client, createInp CreateInput) (*CreateOutput, error) {
+
+	client.Logger.Println("creating cloud ftd onboarding")
+
+	// 1. read fmc domain uid
+	client.Logger.Println("retrieving fmc domain uid")
+
+	// 1.1 read fmc
+	fmcRes, err := cloudfmc.Read(ctx, client, cloudfmc.NewReadInput())
+	if err != nil {
+		return nil, err
+	}
+	// 1.2 read fmc domain info in its specific/appliance device
```

Comment:
I can't understand this comment. What is the difference between ReadFmcDomainInfo and NewReadDomainInfo?

Do you mean to say that you're reading the FMC Domain UID for this FMC appliance?

Reply:
Ah sorry, it is suppose to mean `NewReadDomainInfoInput` instead of `NewReadDomainInfo`, renamed.